src/runtime_src/core/tools/xbreplay/src/main.cpp

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#include "seq_reconstructor.hpp"
+#include "utils/cmd_args.hpp"
+#include "utils/logger.hpp"
+
+#include <iostream>
+#include <memory>
+namespace {
+using namespace xrt::tools::xbreplay;
```

Comment:
Using `using` is not great, it makes reading the code harder because you don't know where the symbol is coming from.
Your code should already be under `namespace xrt_core::tools` so referencing symbols under `xbreplay` should be done as `xbreplay::symbol(...)`

However you are coding in the unnamed namespace (which is good) and a namespace alias can be appropriate to save typing, something like: `namespace xbr = xrt_core::tools::xbreplay`